solr/core/src/test/org/apache/solr/handler/TestSolrConfigHandlerConcurrent.java

```diff
@@ -97,12 +96,7 @@ public void test() throws Exception {
     assertTrue(collectErrors.toString(), success);
   }
 
-  private void invokeBulkCall(
-      String cacheName,
-      List<String> errs,
-      // TODO this is unused - is that a bug?
```

Comment:
on the basis of this comment, how about excluding this file from the cleanup?

Reply:
I * think* that this todo was introduced in SOLR-15385 RawTypes Part VI, and was a reminder to check.  I ran the tests and they pass with this change...  so based on that, inclined to keep the change and be happy that the TODO can be removed...